instrumentation/jedis-5.0.0/src/main/java/redis/clients/jedis/JedisPubSubBase_Instrumentation.java

```diff
@@ -0,0 +1,76 @@
+package redis.clients.jedis;
+
+import com.newrelic.agent.bridge.datastore.DatastoreVendor;
+import com.newrelic.api.agent.DatastoreParameters;
+import com.newrelic.api.agent.NewRelic;
+import com.newrelic.api.agent.Trace;
+import com.newrelic.api.agent.weaver.MatchType;
+import com.newrelic.api.agent.weaver.Weave;
+import com.newrelic.api.agent.weaver.Weaver;
+
+@SuppressWarnings({ "ResultOfMethodCallIgnored", "unused" })
+@Weave(type = MatchType.ExactClass, originalName = "redis.clients.jedis.JedisPubSubBase")
+public class JedisPubSubBase_Instrumentation {
+
+  private volatile Connection client;
+
+  public final void proceed(Connection client, Object... channels) {
```

Comment:
does this method need instrumentation? There is no added code and it is not called inside this class.

If it is here so this does not apply to a previous version, you can make this class abstract and also this method.

Else the weaver ends up adding some code to the method IIRC.